### PR TITLE
 HSC-24: Added reason for death concepts

### PR DESCRIPTION
**Pull request by reagan-meant**

### CODE REVIEW

File: configuration/concepts/reason_for_death.csv [hunk 1]
```diff
@@ -1,21 +1,25 @@
...
+f94c7e31-8d35-4fcf-9564-f06bceb39d4d,,,Road accidents,Accidents de la route,Road accidents,Accidents de la route,,,Misc,N/A,,,,,,,,,,,
+096fcb21-c2b2-42d2-82ec-258521d5f180,,,Cerebrovascular accident,Accident vasculaire cérébral,Cerebrovascular accident,Accident vasculaire cérébral,,,Misc,N/A,,,,,,,,,,,
+c2e25990-13c2-41db-b91c-a72235614721,,,Asphyxia,Asphyxie,Asphyxia,Asphyxie,,,Misc,N/A,,,,,,,,,,,
+ac2909d0-ba32-4291-b1a8-01e7dc1f5029,,,Other trauma,Autres traumatismes,Other trauma,Autres traumatismes,,,Misc,N/A,,,,,,,,,,,
+5bddb65e-238a-4b94-a6bc-4655838e2c0e,,,Complications of pregnancy or childbirth,Complications de la grossesse ou de l’accouchement,Complications of pregnancy or childbirth,Complications de la grossesse ou de l’accouchement,,,Misc,N/A,,,,,,,,,,,
+93f085f3-a932-4784-a4a8-98448d8bc46a,,,HIV-related complications,Complications liées au VIH,HIV-related complications,Complications liées au VIH,,,Misc,N/A,,,,,,,,,,,
+6a52ba5b-1e48-4278-b9a3-422e7c58ce58,,,Post operative complications,Complications post opératoires,Post operative complications,Complications post opératoires,,,Misc,N/A,,,,,,,,,,,
```
Comment (rbuisson) [inline configuration/concepts/reason_for_death.csv:8]: "post-operative" and "post-opératoire"

**Review by rbuisson — CHANGES_REQUESTED** [review]
See comments.
And let's make another PR to move the file to Haiti.

**Comment by reagan-meant** [comment]
Moved this csv to [Haiti level](https://github.com/mekomsolutions/openmrs-config-haiti/pull/28)

File: configuration/concepts/reason_for_death.csv (hunk 1 above)
Comment (reagan-meant) [reply configuration/concepts/reason_for_death.csv]: Done

File: configuration/concepts/reason_for_death.csv [hunk 2]
```diff
@@ -1,21 +1,25 @@
...
-6a52ba5b-1e48-4278-b9a3-422e7c58ce58,,,Complications post opératoires,Complications post opératoires,Complications post opératoires,Complications post opératoires,,,Misc,N/A,,,,,,,,,,,
-291a1ad3-d3cc-43b6-8c4b-059996eed361,,,Diabète,Diabète,Diabète,Diabète,,,Misc,N/A,,,,,,,,,,,
-1f0d9786-366e-4238-9e0f-b9c4695a570d,,,Diarrhée,Diarrhée,Diarrhée,Diarrhée,,,Misc,N/A,,,,,,,,,,,
+f94c7e31-8d35-4fcf-9564-f06bceb39d4d,,,Road accidents,Accidents de la route,Road accidents,Accidents de la route,,,Misc,N/A,,,,,,,,,,,
+096fcb21-c2b2-42d2-82ec-258521d5f180,,,Cerebrovascular accident,Accident vasculaire cérébral,Cerebrovascular accident,Accident vasculaire cérébral,,,Misc,N/A,,,,,,,,,,,
+c2e25990-13c2-41db-b91c-a72235614721,,,Asphyxia,Asphyxie,Asphyxia,Asphyxie,,,Misc,N/A,,,,,,,,,,,
+ac2909d0-ba32-4291-b1a8-01e7dc1f5029,,,Other trauma,Autres traumatismes,Other trauma,Autres traumatismes,,,Misc,N/A,,,,,,,,,,,
+5bddb65e-238a-4b94-a6bc-4655838e2c0e,,,Complications of pregnancy or childbirth,Complications de la grossesse ou de l’accouchement,Complications of pregnancy or childbirth,Complications de la grossesse ou de l’accouchement,,,Misc,N/A,,,,,,,,,,,
```
[the remark this answers is not in the document]
Comment (reagan-meant) [reply configuration/concepts/reason_for_death.csv]: Done

File: configuration/concepts/reason_for_death.csv [hunk 3]
```diff
@@ -1,21 +1,25 @@
...
+7cc9702d-f367-4367-85f9-72ac4b406c23,,,Endocrinological disorder,Problèmes endocrinologique,Endocrinological disorder,Problèmes endocrinologique,,,Misc,N/A,,,,,,,,,,,
+f7169739-7134-4050-8b69-6f55448693f6,,,Respiratory disorder,Problèmes respiratoire,Respiratory disorder,Problèmes respiratoire,,,Misc,N/A,,,,,,,,,,,
+99ba1b00-280c-470a-8ba0-4f8ef84b10df,,,Trauma at birth,Traumatisme à la naissance,Trauma at birth,Traumatisme à la naissance,,,Misc,N/A,,,,,,,,,,,
+b716b7da-8e76-4805-b76b-2ca4b676cf12,,,Tuberculosis,Tuberculose,Tuberculosis,Tuberculose,,,Misc,N/A,,,,,,,,,,,
+30e1c728-f008-4dd1-ab34-2c34a00fd3a5,,,Reason For Death,Raison du décès,Reason For Death,Raison du décès,,,Misc,N/A,,,Road accidents;Cerebrovascular accident;Asphyxia;Other trauma;Other;Unknown;Complications of pregnancy or childbirth;HIV-related complications; Post operative complications;HTA;Malnutrition;Neonatal pathologies;Cardiovascular disorder;Gastrointestinal disorder;Neurological disorder;Obstetrics disorder;Gynecological disorder;Oncology;Orthopedics disorder;Genitourinary disorder;Hematological disorder;Endocrinological disorder;Respiratory disorder;Trauma at birth;Tuberculosis,,,,,,,,
```
[the remark this answers is not in the document]
Comment (reagan-meant) [reply configuration/concepts/reason_for_death.csv]: Done

File: configuration/concepts/reason_for_death.csv [hunk 4]
```diff
@@ -1,21 +1,25 @@
...
+0819e2d9-4ce4-426f-b151-100c5cddf560,,,Obstetrics disorder,Problèmes obstétriques,Obstetrics disorder,Problèmes obstétriques,,,Misc,N/A,,,,,,,,,,,
+46e62928-7c59-4eb7-a450-7bf5eab1a3d7,,,Gynecological disorder,Problèmes gynécologiques,Gynecological disorder,Problèmes gynécologiques,,,Misc,N/A,,,,,,,,,,,
+7869814e-c6f5-4131-97c4-af0912c499fd,,,Oncology,Oncologie,Oncology,Oncologie,,,Misc,N/A,,,,,,,,,,,
+7e96af55-e5f7-46f7-8024-2ce9cea3022f,,,Orthopedics disorder,Problèmes Orthopédie,Orthopedics disorder,Problèmes Orthopédie,,,Misc,N/A,,,,,,,,,,,
+b3c1d5d0-1a77-416e-98ac-af6fe89b603b,,,Genitourinary disorder,Problèmes génito-urinaire,Genitourinary disorder,Problèmes génito-urinaire,,,Misc,N/A,,,,,,,,,,,
+540533db-182f-47b9-b912-f961decbd253,,,Hematological disorder,Problèmes hématologique,Hematological disorder,Problèmes hématologique,,,Misc,N/A,,,,,,,,,,,
+7cc9702d-f367-4367-85f9-72ac4b406c23,,,Endocrinological disorder,Problèmes endocrinologique,Endocrinological disorder,Problèmes endocrinologique,,,Misc,N/A,,,,,,,,,,,
+f7169739-7134-4050-8b69-6f55448693f6,,,Respiratory disorder,Problèmes respiratoire,Respiratory disorder,Problèmes respiratoire,,,Misc,N/A,,,,,,,,,,,
```
[the remark this answers is not in the document]
Comment (reagan-meant) [reply configuration/concepts/reason_for_death.csv]: Done

File: configuration/concepts/reason_for_death.csv [hunk 5]
```diff
@@ -1,21 +1,25 @@
...
+4434134a-b09e-4177-8c9e-6ac61d836015,,,Neurological disorder,Problèmes neurologiques,Neurological disorder,Problèmes neurologiques,,,Misc,N/A,,,,,,,,,,,
+0819e2d9-4ce4-426f-b151-100c5cddf560,,,Obstetrics disorder,Problèmes obstétriques,Obstetrics disorder,Problèmes obstétriques,,,Misc,N/A,,,,,,,,,,,
+46e62928-7c59-4eb7-a450-7bf5eab1a3d7,,,Gynecological disorder,Problèmes gynécologiques,Gynecological disorder,Problèmes gynécologiques,,,Misc,N/A,,,,,,,,,,,
+7869814e-c6f5-4131-97c4-af0912c499fd,,,Oncology,Oncologie,Oncology,Oncologie,,,Misc,N/A,,,,,,,,,,,
+7e96af55-e5f7-46f7-8024-2ce9cea3022f,,,Orthopedics disorder,Problèmes Orthopédie,Orthopedics disorder,Problèmes Orthopédie,,,Misc,N/A,,,,,,,,,,,
+b3c1d5d0-1a77-416e-98ac-af6fe89b603b,,,Genitourinary disorder,Problèmes génito-urinaire,Genitourinary disorder,Problèmes génito-urinaire,,,Misc,N/A,,,,,,,,,,,
+540533db-182f-47b9-b912-f961decbd253,,,Hematological disorder,Problèmes hématologique,Hematological disorder,Problèmes hématologique,,,Misc,N/A,,,,,,,,,,,
+7cc9702d-f367-4367-85f9-72ac4b406c23,,,Endocrinological disorder,Problèmes endocrinologique,Endocrinological disorder,Problèmes endocrinologique,,,Misc,N/A,,,,,,,,,,,
```
[the remark this answers is not in the document]
Comment (reagan-meant) [reply configuration/concepts/reason_for_death.csv]: Done

File: configuration/concepts/reason_for_death.csv [hunk 6]
```diff
@@ -1,21 +1,25 @@
...
+c15139c6-bd0c-4335-a203-6b54cd10b00f,,,Gastrointestinal disorder,Problèmes gastro-intestinaux,Gastrointestinal disorder,Problèmes gastro-intestinaux,,,Misc,N/A,,,,,,,,,,,
+4434134a-b09e-4177-8c9e-6ac61d836015,,,Neurological disorder,Problèmes neurologiques,Neurological disorder,Problèmes neurologiques,,,Misc,N/A,,,,,,,,,,,
+0819e2d9-4ce4-426f-b151-100c5cddf560,,,Obstetrics disorder,Problèmes obstétriques,Obstetrics disorder,Problèmes obstétriques,,,Misc,N/A,,,,,,,,,,,
+46e62928-7c59-4eb7-a450-7bf5eab1a3d7,,,Gynecological disorder,Problèmes gynécologiques,Gynecological disorder,Problèmes gynécologiques,,,Misc,N/A,,,,,,,,,,,
+7869814e-c6f5-4131-97c4-af0912c499fd,,,Oncology,Oncologie,Oncology,Oncologie,,,Misc,N/A,,,,,,,,,,,
+7e96af55-e5f7-46f7-8024-2ce9cea3022f,,,Orthopedics disorder,Problèmes Orthopédie,Orthopedics disorder,Problèmes Orthopédie,,,Misc,N/A,,,,,,,,,,,
+b3c1d5d0-1a77-416e-98ac-af6fe89b603b,,,Genitourinary disorder,Problèmes génito-urinaire,Genitourinary disorder,Problèmes génito-urinaire,,,Misc,N/A,,,,,,,,,,,
+540533db-182f-47b9-b912-f961decbd253,,,Hematological disorder,Problèmes hématologique,Hematological disorder,Problèmes hématologique,,,Misc,N/A,,,,,,,,,,,
```
[the remark this answers is not in the document]
Comment (reagan-meant) [reply configuration/concepts/reason_for_death.csv]: Done

File: configuration/concepts/reason_for_death.csv [hunk 7]
```diff
@@ -1,21 +1,25 @@
...
+c15139c6-bd0c-4335-a203-6b54cd10b00f,,,Gastrointestinal disorder,Problèmes gastro-intestinaux,Gastrointestinal disorder,Problèmes gastro-intestinaux,,,Misc,N/A,,,,,,,,,,,
+4434134a-b09e-4177-8c9e-6ac61d836015,,,Neurological disorder,Problèmes neurologiques,Neurological disorder,Problèmes neurologiques,,,Misc,N/A,,,,,,,,,,,
+0819e2d9-4ce4-426f-b151-100c5cddf560,,,Obstetrics disorder,Problèmes obstétriques,Obstetrics disorder,Problèmes obstétriques,,,Misc,N/A,,,,,,,,,,,
+46e62928-7c59-4eb7-a450-7bf5eab1a3d7,,,Gynecological disorder,Problèmes gynécologiques,Gynecological disorder,Problèmes gynécologiques,,,Misc,N/A,,,,,,,,,,,
+7869814e-c6f5-4131-97c4-af0912c499fd,,,Oncology,Oncologie,Oncology,Oncologie,,,Misc,N/A,,,,,,,,,,,
+7e96af55-e5f7-46f7-8024-2ce9cea3022f,,,Orthopedics disorder,Problèmes Orthopédie,Orthopedics disorder,Problèmes Orthopédie,,,Misc,N/A,,,,,,,,,,,
+b3c1d5d0-1a77-416e-98ac-af6fe89b603b,,,Genitourinary disorder,Problèmes génito-urinaire,Genitourinary disorder,Problèmes génito-urinaire,,,Misc,N/A,,,,,,,,,,,
```
[the remark this answers is not in the document]
Comment (reagan-meant) [reply configuration/concepts/reason_for_death.csv]: Done

File: configuration/concepts/reason_for_death.csv [hunk 8]
```diff
@@ -1,21 +1,25 @@
...
+e4e090ef-5b00-4afc-85ca-7b9fc8190ce0,,,Neonatal pathologies,Pathologies néonatales,Neonatal pathologies,Pathologies néonatales,,,Misc,N/A,,,,,,,,,,,
+55fadeb8-0e31-49a9-93cf-dce6514a3ea6,,,Cardiovascular disorder,Problèmes cardiovasculaires,Problèmes cardiovasculaires,Problèmes cardiovasculaires,,,Misc,N/A,,,,,,,,,,,
+c15139c6-bd0c-4335-a203-6b54cd10b00f,,,Gastrointestinal disorder,Problèmes gastro-intestinaux,Gastrointestinal disorder,Problèmes gastro-intestinaux,,,Misc,N/A,,,,,,,,,,,
+4434134a-b09e-4177-8c9e-6ac61d836015,,,Neurological disorder,Problèmes neurologiques,Neurological disorder,Problèmes neurologiques,,,Misc,N/A,,,,,,,,,,,
+0819e2d9-4ce4-426f-b151-100c5cddf560,,,Obstetrics disorder,Problèmes obstétriques,Obstetrics disorder,Problèmes obstétriques,,,Misc,N/A,,,,,,,,,,,
+46e62928-7c59-4eb7-a450-7bf5eab1a3d7,,,Gynecological disorder,Problèmes gynécologiques,Gynecological disorder,Problèmes gynécologiques,,,Misc,N/A,,,,,,,,,,,
+7869814e-c6f5-4131-97c4-af0912c499fd,,,Oncology,Oncologie,Oncology,Oncologie,,,Misc,N/A,,,,,,,,,,,
+7e96af55-e5f7-46f7-8024-2ce9cea3022f,,,Orthopedics disorder,Problèmes Orthopédie,Orthopedics disorder,Problèmes Orthopédie,,,Misc,N/A,,,,,,,,,,,
```
[the remark this answers is not in the document]
Comment (reagan-meant) [reply configuration/concepts/reason_for_death.csv]: Done

**Pull request merged**